test/acvp_data/acvp_keygen_internalProjection.json

```diff
@@ -0,0 +1,555 @@
+{
+  "vsId": 42,
+  "algorithm": "ML-DSA",
+  "mode": "keyGen",
+  "revision": "FIPS204",
+  "isSample": false,
+  "testGroups": [
+    {
+      "tgId": 1,
+      "testType": "AFT",
+      "parameterSet": "ML-DSA-44",
+      "tests": [
+        {
+          "tcId": 1,
+          "deferred": false,
```

Comment:
What does this parameter mean?

Reply:
The documentation does not say:  https://pages.nist.gov/ACVP/draft-celi-acvp-ml-dsa.html

Same in mlkem-native: https://github.com/pq-code-package/mlkem-native/blob/ba26e0df4c7c9dc4f4084abd39d23af403d6299b/test/acvp_data/acvp_keygen_internalProjection.json#L15